projects/plugins/jetpack/_inc/lib/class.media-summary.php

```diff
@@ -264,7 +264,7 @@ public static function get( $post_id, $blog_id = 0, $args = array() ) {
 				$return['secure']['image'] = self::ssl_img( $return['image'] );
 				++$return['count']['image'];
 
-				if ( $number_of_paragraphs <= 2 && 1 === count( $extract['image'] ) ) {
+				if ( $number_of_paragraphs <= 2 && is_countable( $extract['image'] ) && 1 === count( $extract['image'] ) ) {
```

Comment:
I note line 263 would also have problems if it's somehow not the expected array, although that line will only warn while this in PHP 8 will throw.